### PR TITLE
MAT 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/cosmic_rebirth.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Cosmic Rebirth
+ManaCost:1 G W
+Types:Instant
+A:SP$ ChangeZone | ValidTgts$ Permanent.YouOwn | TgtPrompt$ Select target permanent card in your graveyard | Origin$ Graveyard | Destination$ Battlefield | ConditionDefined$ Targeted | ConditionPresent$ Card.cmcLE3 | SubAbility$ DBChangeZone | SpellDescription$ Choose target permanent card in your graveyard. If it has mana value 3 or less, you may put it onto the battlefield. If you don't put it onto the battlefield, put it into your hand.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/cosmic_rebirth.txt:4]: I don't think the "may" part is handled correctly

File: forge-gui/res/cardsfolder/upcoming/harnessed_snubhorn.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Harnessed Snubhorn
+ManaCost:3 W
+Types:Creature Dinosaur
+PT:2/5
+K:Vigilance
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player,Battle | Execute$ TrigReturn | CombatDamage$ True | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, return target artifact or enchantment card from your graveyard to the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/harnessed_snubhorn.txt:6]: wrong Target?

File: forge-gui/res/cardsfolder/upcoming/harnessed_snubhorn.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Harnessed Snubhorn
+ManaCost:3 W
+Types:Creature Dinosaur
+PT:2/5
+K:Vigilance
+T:Mode$ DamageDone | ValidSource$ Card.Self | Optional$ True | ValidTarget$ Player | Execute$ TrigReturn | CombatDamage$ True | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, return target artifact or enchantment card from your graveyard to the battlefield.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/harnessed_snubhorn.txt:6]: Optional here doesn't make sense but maybe you wanted to put it on the other card?

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/harnessed_snubhorn.txt:6]: Yeah

**Review by tool4ever — APPROVED** [review]

**Pull request merged**